cli/test/datafeed.test.ts

```diff
@@ -0,0 +1,154 @@
+  })
+
+  afterAll(async () => {
+    const chains = await chainListHandler()()
+    for (const chain of chains) {
+      await chainRemoveHandler()({ id: chain.id })
+    }
+    const services = await serviceListHandler()()
+    for (const service of services) {
+      await serviceRemoveHandler()({ id: service.id })
+    }
+    const organizations = await organizationListHandler()()
+    for (const organization of organizations) {
+      await organizationRemoveHandler()({ id: organization.id })
+    }
+  })
+
+  afterEach(async () => {
+    const afterAdapterList = await adapterListHandler()()
+    const afterAggregatorList = await aggregatorListHandler()({})
+    const afterDelegatorReporterList = await delegatorReporterListHandler()()
+    const afterContractList = await contractListHandler()()
+    const afterListenerList = await listenerListHandler()({})
+    const afterReporterList = await reporterListHandler()({})
+    const afterFunctionList = await functionListHandler()()
+
+    for (const _function of afterFunctionList) {
+      await functionRemoveHandler()({ id: Number(_function.id) })
+    }
+    for (const aggregator of afterAggregatorList) {
+      await aggregatorRemoveHandler()({ id: aggregator.id })
+    }
+    for (const adapter of afterAdapterList) {
+      await adapterRemoveHandler()({ id: adapter.id })
+    }
+    for (const delegatorReporter of afterDelegatorReporterList) {
+      await delegatorReporterRemoveHandler()({ id: delegatorReporter.id })
+    }
+    for (const contract of afterContractList) {
+      await contractRemoveHandler()({ id: contract.id })
+    }
+    for (const listener of afterListenerList) {
+      await listenerRemoveHandler()({ id: listener.id })
+    }
+    for (const reporter of afterReporterList) {
+      await reporterRemoveHandler()({ id: reporter.id })
+    }
+  })
+
+  test('datafeed bulk insert with default values', async function () {
+    const beforeAdapterList = await adapterListHandler()()
+    const beforeAggregatorList = await aggregatorListHandler()({})
+    const beforeDelegatorReporterList = await delegatorReporterListHandler()()
+    const beforeContractList = await contractListHandler()()
+    const beforeListenerList = await listenerListHandler()({})
+    const beforeReporterList = await reporterListHandler()({})
+    const beforeFunctionList = await functionListHandler()()
+
+    await bulkInsertHandler()({ data: DATAFEED_BULK_0 })
+
+    const afterAdapterList = await adapterListHandler()()
+    const afterAggregatorList = await aggregatorListHandler()({})
+    const afterDelegatorReporterList = await delegatorReporterListHandler()()
+    const afterContractList = await contractListHandler()()
+    const afterListenerList = await listenerListHandler()({})
+    const afterReporterList = await reporterListHandler()({})
+    const afterFunctionList = await functionListHandler()()
+
+    expect(afterAdapterList.length).toEqual(beforeAdapterList.length + 3)
```

Comment:
Since `DATAFEED_BULK_0` is defined in different file and it is not obvious where the 3 comes from, I would recommend to not use 3 here, but rather get the length of bulk. This applies to following tests as well.

```suggestion
cost bulkLength = DATAFEED_BULK_0.length
expect(afterAdapterList.length).toEqual(beforeAdapterList.length + bulkLength)
```